src/tests/regress/data/sql/interval.sql

```diff
@@ -111,17 +111,17 @@ INSERT INTO INTERVAL_MULDIV_TBL VALUES
 ('14 mon'),
 ('999 mon 999 days');
 
---@ SELECT span * 0.3 AS product
---@ FROM INTERVAL_MULDIV_TBL;
---@ 
---@ SELECT span * 8.2 AS product
---@ FROM INTERVAL_MULDIV_TBL;
---@ 
---@ SELECT span / 10 AS quotient
---@ FROM INTERVAL_MULDIV_TBL;
---@ 
---@ SELECT span / 100 AS quotient
---@ FROM INTERVAL_MULDIV_TBL;
+SELECT span * 0.3 AS product
+FROM INTERVAL_MULDIV_TBL;
+
+SELECT span * 8.2 AS product
+FROM INTERVAL_MULDIV_TBL;
+
+SELECT span / 10 AS quotient
+FROM INTERVAL_MULDIV_TBL;
+
+SELECT span / 100 AS quotient
+FROM INTERVAL_MULDIV_TBL;
```

Comment:
Actually how are these tested? Where is the expected output? Or do regress tests just check if the DBMS crashed?

Reply:
https://github.com/risingwavelabs/risingwave/tree/main/src/tests/regress
The expected output is at
https://github.com/risingwavelabs/risingwave/tree/main/src/tests/regress/data/expected

The test suits are taken from PostgreSQL.